app/search/models.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import re
+import sqlalchemy
+import json
+from app.package.models import Package
+from app.profile.models import Publisher
+
+
+class DataPackageQuery(object):
+
+    query = ''
```

Comment:
But what about risk of multiple calls to class at the same time? I'm not sure i understand the logic here ... Even if merge this can you explain more here.